src/shared/components/query/QueryStore.ts

```diff
@@ -1535,6 +1539,51 @@ export class QueryStore {
         return result;
     }
 
+    @computed get defaultMutationProfile() {
```

Comment:
seems like there is a lot of repetition here.  why not have defaultProfiles object populate and extract out the "find" logic into a function.